mockk/jvm/src/test/kotlin/io/mockk/junit4/MockKRuleTest.kt

```diff
@@ -0,0 +1,85 @@
+@file:Suppress("UNUSED_PARAMETER")
+
+package io.mockk.junit4
+
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.impl.annotations.SpyK
+import io.mockk.verify
+import org.junit.Rule
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
```

Comment:
You need `RunWith(JUnit4::class)` here for the tests to pass

Reply:
Updated